pkg/resource/user/hooks.go

```diff
@@ -55,6 +59,14 @@ func (rm *resourceManager) validateUserNeedsUpdate(
 	return nil, nil
 }
 
+// userActive returns true when the status of the given User is set to `active`
+func (rm *resourceManager) userActive(
```

Comment:
nit: methods returning booleans should start with an article. For example `isUserActive`

Reply:
@kyriechen96 wants to address this in a seperate PR